ssm/agents.py

```diff
@@ -259,6 +265,13 @@ def run_sender(protocol, brokers, project, token, cp, log):
 
 def run_receiver(protocol, brokers, project, token, cp, log, dn_file):
     """Run Ssm2 as a receiver daemon."""
+    if DaemonContext is None:
```

Comment:
I moved to this the equivalent place in `agents.py` following the rebase. Before the rebase, it would have been nice to run this earlier in `receiver.py` (to avoid some of the setup sets which would be pointless if python-daemon isn't available), but that doesn't make sense following the rebase so here it shall stay!